pandas/core/frame.py

```diff
@@ -1720,7 +1720,7 @@ def from_dict(
             if columns is not None:
                 raise ValueError(f"cannot use columns parameter with orient='{orient}'")
         else:  # pragma: no cover
-            raise ValueError("only recognize index or columns for orient")
+            raise ValueError("only recognize 'index', 'columns', or 'tight' for orient")
```

Comment:
We should add a test checking for this error message

Reply:
Yup, sounds good. I've updated the error msg